openedx/features/edly/api/serializers.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import serializers
+
+from openedx.core.djangoapps.site_configuration.helpers import get_value_for_org
+
+
+class UserSiteSerializer(serializers.Serializer):
+    app_config = serializers.SerializerMethodField()
+    site_data = serializers.SerializerMethodField()
+
+    def get_app_config(self, obj):
+        mobile_app_config = get_value_for_org(
+            self.context['edly_sub_org_of_user'].edx_organization.short_name,
+            'MOBILE_APP_CONFIG',
+            default={}
+        )
+        url = get_value_for_org(
+            self.context['edly_sub_org_of_user'].edx_organization.short_name,
+            'SITE_NAME',
+            default=''
+        )
+        protocol = 'https' if self.context['request'].is_secure() else 'http'
+        mobile_app_config['API_HOST_URL'] = '{}//:{}'.format(protocol, url) if url else ''
```

Comment:
@iam-mhaseeb url should be in right format

Reply:
The fix is happening here
https://github.com/edly-io/edx-platform/pull/159